Packs/Azure/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "Azure",
     "description": "Microsoft Azure",
     "support": "xsoar",
-    "currentVersion": "1.0.7",
+    "currentVersion": "1.0.9",
```

```suggestion
    "currentVersion": "1.0.9",
```
Why is it not 1.0.8?

---

I added the changes to 1_1_0
As we are merging to dev-cloud, the release notes will be merged to 1